CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,31 @@
+•  Be patient
+
+Not everyone has the same level of expertise in language or coding, so be patient if an explanation needs to be given.
+
+•  Be welcoming
+
+Aim to welcome and support people of all backgrounds and identities, of any race, ethnicity, culture, national origin, color, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
+
+•  Be respectful and friendly
+
+Respect each other and each other's point of view. Different people have different perspectives on issues. Frustration cannot turn into a personal attack. It is important to remember that a community where people feel uncomfortable or threatened is not a productive one. Also remember that with a world-wide community, English may not be the home language of everyone and since all communication happens via text, it's impossible to read what mood the other is in.

```

Comment:
Can we change "home language" to read "first language" or "native language"?

Reply:
Okay, I can live with that. I'll change it to 'native language'.

Reply:
Thanks, I see the commit below.